docs/flow/modin/core/dataframe/algebra.rst

```diff
@@ -90,7 +90,14 @@ equals to the number of CPUs so that each single axis partition gets processed i
 GroupBy operator
 ----------------
 Evaluates GroupBy aggregation for that type of functions that can be executed via TreeReduce approach.
-To be able to form groups engine broadcasts ``by`` partitions to each partition of the source frame.
+To be able to form groups engine broadcasts ``by`` partitions to each partition of the source frame or
+applies range partitioning approach.
```

Comment:
What we're supposed to describe here? If we're talking about [`GroupByReduce`](https://github.com/modin-project/modin/blob/199e3fd3cfa4f03fae107f213dfb68781562bc93/modin/core/dataframe/algebra/groupby.py#L26) operator, then the line regarding range-partitioning approach is false since it doesn't do so.